### ISSUE TITLE
Core Importer forms select from default schema for form elements -should it be the schema chosen for import?

### ISSUE THREAD

**Issue by risharde** [tripal/tripal#1358]
Because we now have the ability to run database operations of different schemas, it might be a good idea to also make the loader forms dynamic with respect to the chado schema element and thus depending elements. 

There are fields within these loaders forms which should be populated based on the schema chosen. For example, the select list for organism and analysis (this is not an extensive list) should show whatever records are available from the specified schema - so putting the schema name element above would probably be a good workflow for the rest of dependant ui form elements.

Tagging @spficklin since this was brough up in our weekly check-in meeting. Tagging @laceysanderson as well for any additional insight and contributions where necessary.

**Comment by laceysanderson** [comment]
Yes, good point! I agree with this suggestion. This is already in the ChadoImporterBase::form() so that specific loaders to do not need to worry about it. Then the ChadoImporterBase::getChadoConnection() uses the schema set/selected. As long as we set `$this->chado_schema_main` via AJAX when someone selects a different schema, then they will get the right schema when they call getChadoConnection(). 

I think if we put it at the top we should theme it so it looks more like a sub-title/information element. Then, as long as we select the default in a smart manner, any sites with a single chado can basically ignore it.

**Comment by laceysanderson** [comment]
Update for anyone who comes across this...
The chado schema form element is still at the bottom of importer forms in an "advanced" fieldset... 

The core importers select any data for select lists, etc from the "Default" chado rather than worrying about ensuring they select from the schema being inserted into.

This issue is being left open so if you do have a site using multiple chado schema and are finding this not helpful -please comment here so we can up the priority.
